engine/tests/phpunit/integration/Elgg/Integration/ElggCoreEntityTest.php

```diff
@@ -189,7 +189,7 @@ public function testElggEntitySaveAndDelete() {
 	}
 
 	public function testElggEntityDisableAndEnable() {
-		$CONFIG = _elgg_config();
+		$CONFIG = _elgg_services()->config;
```

Comment:
is `$CONFIG` really needed or can this be fixed in a different way (`elgg()->db->prefix`)